### PR TITLE
Add LIFCL-17 SYSIO tiles to the 070 fuzzer

### PR DESCRIPTION
**Pull request by mkurc-ant**
This PR enables `070-iologic_mode` fuzzer to operate on `LIFCL-17` SYSIO tiles so that bits controlling eg. `IDDRX1` and `ODDRX1` for them are solved too.

The tiles included are:
 - SYSIO_B0_0_15K
 - SYSIO_B1_0_15K
 - SYSIO_B1_DED_15K (the fuzzer does not solve the mode bits for this one)

There are also SYSIO_B1_1_15K in `LIFCL-17` but they don't have their pads exposed so that they cannot be used.

### CODE REVIEW

**Comment by mkurc-ant** [comment]
Looking throughout  the `prjoxide` database for bits controlling the `IDDRX1.ODDRX1` mode revealed that they are not defined for the following tiles:
- SYSIO_B0_0_15K*
- SYSIO_B1_0_15K*
- SYSIO_B1_0_REM
- SYSIO_B1_1_15K
- SYSIO_B1_DED_15K*
- SYSIO_B2_0_REM
- SYSIO_B6_0_REM
- SYSIO_B7_0_REM

The ones with `*` should be solved by `070-iologic_mode` with this PR. Not sure though if the rest of them actually have the `IOLOGIC` inside.

File: fuzzers/LIFCL/070-iologic_mode/fuzzer.py [hunk 1]
```diff
@@ -46,30 +48,82 @@
 
     ("IOL_R3B", "SIOLOGICB", FuzzConfig(job="IOL3DEMODE", device="LIFCL-40", sv="../shared/empty_40.v", tiles=["CIB_R3C87:SYSIO_B1_DED"])),
 
+    # LIFCL-17 tiles
+    ("IOL_T57A", "SIOLOGICA", FuzzConfig(job="IOLT57AMODE", device="LIFCL-17", sv="../shared/empty_17.v", tiles=["CIB_R0C57:SYSIO_B0_0_15K"])),
```
Comment (gatecat) [inline fuzzers/LIFCL/070-iologic_mode/fuzzer.py:52]: the tiletypes here don't look correct; did you actually have a go at running the fuzzer?

Comment (mkurc-ant) [reply fuzzers/LIFCL/070-iologic_mode/fuzzer.py:52]: I'm pretty sure that they are or. I took them from the docs: https://gatecat.github.io/prjoxide-html/LIFCL/LIFCL-17/index.html.

I was able to get solution for `SYSIO_B0_0_15K` and `SYSIO_B1_0_15K`

Comment (gatecat) [reply fuzzers/LIFCL/070-iologic_mode/fuzzer.py:52]: oh, I see, you've added multiple tiles of the same type? that shouldn't be necessary, I think

Comment (mkurc-ant) [reply fuzzers/LIFCL/070-iologic_mode/fuzzer.py:52]: Right, I wasn't sure what's the minimum number of tiles required. What do you suggest? I added all of them to catch possible discrepancies during fuzzing.

Comment (gatecat) [reply fuzzers/LIFCL/070-iologic_mode/fuzzer.py:52]: For bitstream config, only one set of tiles is always safe, and to minimise fuzzer runtime is the way to go (occasionally this is different for routing but that doesn't apply here).

Once this is sorted I'll give it a run, push the new DB and merge.

Comment (mkurc-ant) [reply fuzzers/LIFCL/070-iologic_mode/fuzzer.py:52]: Ok. I left only one tile of each type. I'm getting solution for "regular" `SYSIO_*_15K` but unfortunately still not for `SYSIO_B1_DED_15K`.

Comment (gatecat) [reply fuzzers/LIFCL/070-iologic_mode/fuzzer.py:52]: Thanks, I'll have a look at how it runs now.

Comment (gatecat) [reply fuzzers/LIFCL/070-iologic_mode/fuzzer.py:52]: Looking at unknown bits in a design that uses an ODDR on that pin, it seems like like these bits end up in `CIB_R4C75:PIC_B1_DED_15K`, I've updated the fuzzer accordingly and will see how it goes.

**Comment by DurandA** [comment]
@gatecat Is the [Current Status](https://github.com/gatecat/prjoxide#current-status) section of the README still up-to-date?

**Pull request merged**